action/api/src/main/java/io/knotx/fragments/action/api/log/ActionLog.java

```diff
@@ -48,14 +49,17 @@ public ActionLog(String alias, JsonObject logs, List<ActionInvocationLog> doActi
   }
 
   public ActionLog(JsonObject actionLog) {
-    this.alias = actionLog.getString("alias");
-    this.logs = actionLog.getJsonObject("logs");
+    this.alias = actionLog.containsKey("alias") ? actionLog.getString("alias") : "";
```

Comment:
Experienced `NullPointerException` here. Probably due to introduced `FragmentResult` factory methods `success` and `fail` which make use of `new JsonObject()` instead of `null` for log entry